config.json

```
@@ -2380,6 +2380,21 @@
         ],
         "difficulty": 9
       },
+      {
+        "slug": "satellite",
+        "name": "Satellite",
+        "uuid": "a07e01e6-7cea-4db9-a514-7c95788e90a9",
+        "prerequisites": [
+          "pattern-matching",
+          "tuple",
+          "cond",
+          "recursion"
```

Let's add:
- `atoms` and `lists` because arguments passed to `build_tree` are lists of atoms
- `enum` for a standard uniqueness check (`(list -- Enum.uniq(list)) == []`) that doesn't require knowing about `MapSet`
```suggestion
          "recursion",
          "atoms",
          "lists",
          "enum"
```

---

So many typos... -_- 

I wonder about atoms, because there is no need to handle what the list contains, the tests use atoms, but it could be anything else.

---

Right, but there's also the necessity of returning either `:ok` or `:error`, so the student solving the exercise must know how to write an atom, and that's taught in the `atoms` concept. In practice, having `atoms` as a prerequisite doesn't change anything because `atoms` are a direct prerequisite of `tuples`, and an indirect prerequisite of almost everything else :)